src/Features/CSharp/Portable/Completion/CompletionProviders/TupleNameCompletionProvider.cs

```diff
@@ -75,10 +79,16 @@ private void AddItems(ImmutableArray<INamedTypeSymbol> inferredTypes, int index,
                 }
 
                 var field = type.TupleElements[index];
-                var item = CommonCompletionItem.Create(
-                    field.Name + ":", _cachedRules, Glyph.FieldPublic);
+                var item = CommonCompletionItem.Create(field.Name + ColonString, _cachedRules, Glyph.FieldPublic);
```

Comment:
Added the filterText. 
Would you know how to validate? (the tests I added for exact match don't behave any differently than regular scenarios)